nodetypes/radon.nodes.monitoring/PushGateway/NodeType.tosca

```diff
@@ -0,0 +1,92 @@
+################################################################################
+#                                                                              #
+#   This is the TOSCA node definition for the Prometheus Pushgateway instance. #
```

Comment:
I suggest moving this into the respective README file. AFAIK such info will be overwritten by the GMT when processing the type.

Reply:
I agree

Reply:
Done and commited